launch-game.sh

```diff
@@ -25,11 +25,22 @@ mono OpenRA.Game.exe Engine.LaunchPath="$MODLAUNCHER" $MODARG "$@"
 
 # Show a crash dialog if something went wrong
 if [ $? != 0 ] && [ $? != 1 ]; then
-	ERROR_MESSAGE="OpenRA has encountered a fatal error.\nPlease refer to the crash logs and FAQ for more information.\n\nLog files are located in ~/.openra/Logs\nThe FAQ is available at http://wiki.openra.net/FAQ"
+	# Match with X${OSTYPE} to silence a shell-check warning because ${OSTYPE} is not defined in posix
+	if expr match "X${OSTYPE}" "XDarwin.*" >/dev/null; then
```

Comment:
After all that hassle, this doesn't actually work on macos!

It looks like the "match" argument is a non-standard GNUism that isn't supported by macOS's BSD `expr` :cry:

Reply:
We should be able to use `if [ "$(uname -s)" = "Darwin" ]; then` instead.